droidlet/memory/memory_filters.py

```diff
@@ -471,10 +471,12 @@ def handle_output(self, agent_memory, query, memids):
                         )
                     )
                 for m in memids:
-                    values_dict[m].append(get_property_value(agent_memory, m, aname))
-            if len(attribute_name_list) == 1:
-                for m in values_dict:
-                    values_dict[m] = values_dict[m][0]
+                    prop_vals = get_property_value(agent_memory, m, aname, get_all=True)
+                    if type(prop_vals) is list:
+                        for prop_val in prop_vals:
+                            values_dict[m].append(prop_val)
+                    else:
+                        values_dict[m].append(prop_vals)
             # TODO switch everything to dicts
             return [values_dict[m] for m in memids]
```

Comment:
this needs to be finessed- we should probably be returning a separate record for each value.  so if a memid m has multiple associated properties p0, p1, p2, we should eventually return (from .search()) [m, m, m], [p0, p1, p2]

Reply:
@aszlam see recent commits. This format breaks crafassist unit test when the vals are (x,y) coords:

 ```
query = "SELECT (x, y) FROM ReferenceObject WHERE ((has_tag=plays_volleyball) AND (x<0))"
memids, vals = m.search(self.memory, query=query)
>       assert abs(vals[0][0] + 2.0) < 0.01
E       TypeError: 'float' object is not subscriptable
```

I need to think about how to fix this.